makefile

```
@@ -26,7 +26,7 @@ package:: create-package-folder package-rpm-amd64 package-rpm-386 package-rpm-ar
 release:: clean checkstyle release-test pre-release build prepack package copy-package-dependencies
 
 clean:: remove-prepacked-folder
-	rm -rf build/* bin/ pkg/ vendor/bin/ vendor/pkg/ .cover/
+	rm -rf build/* bin/ pkg/ .cover/
```

```suggestion
	rm -rf build/* bin/ .cover/
```
It looks like some files that shouldn't be cleaned up have been moved into `pkg`